apps/frontend/src/pages/people/index.tsx

```diff
@@ -204,14 +232,219 @@ const PeoplePage: React.FC = () => {
+                  {Object.values(searchResults.bios).map(
+                    (connection, index) => (
+                      <ContactListItem
+                        key={connection.user.username}
+                        connection={connection}
+                        index={index}
+                        darkTheme={darkTheme}
+                      />
+                    )
+                  )}
+                </ul>
+              </section>
+            )}
+
+            {Object.keys(searchResults.notes).length > 0 && (
+              <section>
+                <h2 className="text-label-secondary mb-2">Notes</h2>
+                <div
+                  className="h-[2px]"
+                  style={{
+                    background: `linear-gradient(90deg, #7A74BC 0%, #FF9DF8 39%, #FB5D42 71%, #F00 100%)`,
+                  }}
+                ></div>
+                <ul className="flex flex-col">
+                  {Object.values(searchResults.notes).map(
+                    (connection, index) => (
+                      <ContactListItem
+                        key={connection.user.username}
+                        connection={connection}
+                        index={index}
+                        darkTheme={darkTheme}
+                      />
+                    )
+                  )}
+                </ul>
+              </section>
+            )}
+
+            {Object.keys(searchResults.labels).length > 0 && (
+              <section>
+                <h2 className="text-label-secondary mb-2">Emoji</h2>
+                <div
+                  className="h-[2px]"
+                  style={{
+                    background: `linear-gradient(90deg, #7A74BC 0%, #FF9DF8 39%, #FB5D42 71%, #F00 100%)`,
+                  }}
+                ></div>
+                <ul className="flex flex-col">
+                  {Object.values(searchResults.labels).map(
+                    (connection, index) => (
+                      <ContactListItem
+                        key={connection.user.username}
+                        connection={connection}
+                        index={index}
+                        darkTheme={darkTheme}
+                      />
+                    )
+                  )}
+                </ul>
+              </section>
+            )}
+          </div>
+        )}
+      </AppLayout>
+    );
+  }
+
   return (
     <AppLayout
       seoTitle="People"
       header={
         <div className="flex flex-col">
-          <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4">
-            {`Connections (${Object.keys(connections)?.length})`}
-          </span>
+          <div className="flex flex-row items-center justify-between">
+            <span className="text-label-primary text-xl leading-none font-bold tracking-[-0.1px] py-4">
+              {`Connections (${Object.keys(connections)?.length})`}
+            </span>
+            <div className="ml-auto">
```

Comment:
In the designs the search icon is right-aligned: 
![image](https://github.com/user-attachments/assets/28b39292-3e38-4c1c-bbd8-5ed7d1598187)

Reply:
I solved the *same* styling issue here using `marginLeft: auto`: https://github.com/cursive-team/connections/pull/240/commits/324436c393bf9015e4e4ceed62133bf81416942f 

If you know a tailwind class for auto margin I'd love it.

Reply:
should be `ml-auto`

Reply:
ah makes sense, thanks

Reply:
i've tried both of these, no luck